gobblin-scheduler/src/main/java/gobblin/scheduler/SchedulerDaemon.java

```diff
@@ -72,20 +72,22 @@ public SchedulerDaemon(Properties defaultProperties, Properties customProperties
     List<Service> services = Lists.<Service>newArrayList(new JobScheduler(properties));
     boolean jobExecInfoServerEnabled = Boolean
         .valueOf(properties.getProperty(ConfigurationKeys.JOB_EXECINFO_SERVER_ENABLED_KEY, Boolean.FALSE.toString()));
+    boolean adminUiServerEnabled = Boolean
+        .valueOf(properties.getProperty(ConfigurationKeys.ADMIN_SERVER_ENABLED_KEY, Boolean.FALSE.toString()));
     if (jobExecInfoServerEnabled) {
+      LOG.info("Starting the job execution info server since it is enabled");
       JobExecutionInfoServer executionInfoServer = new JobExecutionInfoServer(properties);
       services.add(executionInfoServer);
-      if (shouldRunAdminServer(properties)) {
+      if (adminUiServerEnabled) {
```

Comment:
for now it's fine. The minor disadvantage is that in the future, if the logic for `shouldRunAdminServer` changes (e.g. inferred from other properties), it would be nicer to have this as a method. But if that is ever the case, we can refactor then.